relay-server/src/services/project.rs

```diff
@@ -1246,6 +1246,7 @@ fn is_metric_namespace_valid(state: &ProjectState, namespace: &MetricNamespace)
             state.has_feature(Feature::ExtractSpansAndSpanMetricsFromEvent)
                 || state.has_feature(Feature::StandaloneSpanIngestion)
         }
+        MetricNamespace::Profiles => true,
```

Comment:
I'd be more comfortable if the profiles were gated by a feature flag, a feature flag we can only enable in S4S first.

Reply:
@Dav1dde I've added the kill-switch option instead, as we discussed offline.